gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```

Comment:
just bump to 6.8 i guess

Reply:
I’d prefer to leave it be, I doubt there will be any issues but it’s easier to track down if updating this stuff is done on its own